jin/3주차 과제/3-기초데이터형.py

```diff
@@ -0,0 +1,11 @@
+# 정수 1개를 입력받아 그대로 출력
+integer = int(input())
+print(integer)
+
+# 실수 1개를 입력받아 그대로 출력
+fl = float(input())
+print(fl)
```

Comment:
```suggestion
print(float(input())
```
이렇게 쓰는 것도 괜찮을 것 같아요~